### DISCUSSION TITLE
Export and importing Tarso model

### DISCUSSION THREAD

**Discussion by strijkerb** [pastas/pastas#564] [Q&A] [answered]
Hi everyone, I have some difficulties with importing a Tarso model. When I export a Tarso model that is solved using `mlt.to_file("file.pas")` and try to import it again using `ps.io.load("file.pas")`, I get the following error message:

`Exception: Please specify either oseries or dmin and dmax`

I also tried to delete the oseries or dmin and max from the TarsoModel, but then I received an error when exporting the model: 
`'TarsoModel' object has no attribute 'oseries'` or `'TarsoModel' object has no attribute 'dmin'`

When I export a linear recharge model with e.g. an exponential response function, the importing process works fine. 

I hope someone can give some suggestions! Regards, Bart

**Comment by rubencalje** [comment, marked as answer]
Hi @strijkerb,

It seems this problem happens when the TarsoModel was created using 
`sm = ps.TarsoModel(prec, evap, oseries=oseries)`

There seems to be no problem when you create the TarsoModel using (which results in the same model-simulation):
`sm = ps.TarsoModel(prec, evap, dmin=oseries.min(), dmax=oseries.max())`

So until we solve this issue that would be a possible solution...

**Reply by rubencalje** [reply]
I will add a fix in the dev-branch of pastas, so this but will not take place in the next version of pastas anymore.
